From: Yajun Deng <yajun.deng@linux.dev>
Subject: mm/mmap: use SZ_{8M, 128M} helper macro
Date: Fri, 26 Jan 2024 16:59:05 +0800

Use SZ_{8M, 128M} macro intead of the number in init_user_reserve and
reserve_mem_notifier.

Link: https://lkml.kernel.org/r/20240126085905.2835513-1-yajun.deng@linux.dev
Signed-off-by: Yajun Deng <yajun.deng@linux.dev>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/mmap.c |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/mm/mmap.c~mm-mmap-use-sz_8m-128m-helper-macro
+++ a/mm/mmap.c
@@ -3847,7 +3847,7 @@ static int init_user_reserve(void)
 
 	free_kbytes = K(global_zone_page_state(NR_FREE_PAGES));
 
-	sysctl_user_reserve_kbytes = min(free_kbytes / 32, 1UL << 17);
+	sysctl_user_reserve_kbytes = min(free_kbytes / 32, SZ_128M);
 	return 0;
 }
 subsys_initcall(init_user_reserve);
@@ -3868,7 +3868,7 @@ static int init_admin_reserve(void)
 
 	free_kbytes = K(global_zone_page_state(NR_FREE_PAGES));
 
-	sysctl_admin_reserve_kbytes = min(free_kbytes / 32, 1UL << 13);
+	sysctl_admin_reserve_kbytes = min(free_kbytes / 32, SZ_8M);
 	return 0;
 }
 subsys_initcall(init_admin_reserve);
@@ -3900,12 +3900,12 @@ static int reserve_mem_notifier(struct n
 	case MEM_ONLINE:
 		/* Default max is 128MB. Leave alone if modified by operator. */
 		tmp = sysctl_user_reserve_kbytes;
-		if (0 < tmp && tmp < (1UL << 17))
+		if (tmp > 0 && tmp < SZ_128M)
 			init_user_reserve();
 
 		/* Default max is 8MB.  Leave alone if modified by operator. */
 		tmp = sysctl_admin_reserve_kbytes;
-		if (0 < tmp && tmp < (1UL << 13))
+		if (tmp > 0 && tmp < SZ_8M)
 			init_admin_reserve();
 
 		break;
_
